javascript/javascript1/week3/smartphone.js

```diff
@@ -0,0 +1,47 @@
+////////////////////Adding activities////////////////////
+const activities = [];
+
+function addActivity(date, activity, duration) {
+  activities.push({
+    date: date,
+    activity: activity,
+    duration: duration
+  });
+}
+addActivity("23/7-18", "Youtube", 30);
+addActivity("22/7-18", "Facebook", 40);
+addActivity("22/7-18", "Whatsapp", 50);
+console.log(activities);
+
+///////////////////////Show my status///////////////////////////////
+let totalDuration = 0;
```

Comment:
if this is only used inside a local function scope, i would consider declaring the variable within the function for two reasons:

1. This informs others(and yourself) that this variable is only relevant within the scope if the function.
2.  App footprint gains by only holding what you need in the memory - global variables are kept in memory where local variables are garbage collected whenever they are no longer used.

Reply:
Got it. Now declared inside the function.